tests/hazmat/bindings/test_openssl.py

```diff
@@ -85,6 +85,61 @@ def test_ssl_mode(self):
         assert resp == expected_options
         assert b.lib.SSL_get_mode(ssl) == expected_options
 
+    @pytest.mark.skipif(
+        Binding.lib.Cryptography_HAS_TLS_METHOD == 0,
+        reason="TLS_method requires OpenSSL >= 1.1.0",
+    )
+    def test_tls_ctx_options(self):
```

Comment:
Duplicating these tests just to pass TLS_method isn't necessary.

Reply:
Noted in the commit update (currently building).  Thanks for catching that.